infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java

```diff
@@ -70,7 +70,7 @@ public enum SystemSchemaBuilderRule {
             "socket_summary_by_event_name", "socket_summary_by_instance", "status_by_account", "status_by_host", "status_by_thread", "status_by_user", "table_handles",
             "table_io_waits_summary_by_index_usage", "table_io_waits_summary_by_table", "table_lock_waits_summary_by_table", "threads", "user_variables_by_thread", "users", "variables_by_thread"))),
     
-    MYSQL_SYS("MySQL", "sys", new HashSet<>(Collections.singleton("sys"))),
+    MYSQL_SYS("MySQL", "sys", new HashSet<>(new HashSet<>(Arrays.asList("sys", "host_summary_by_statement_type")))),
```

Comment:
Why it has two hashset?

Reply:
Sorry for being careless, it has been corrected :astonished: